tests/npm-tsoa-app/README.md

```diff
@@ -0,0 +1,19 @@
+# Node.js/npm CCF app using tsoa
```

Comment:
I would prefer to have this integrated in the main Sphinx doc, where it's much more easily searchable and cross-referenced.

Reply:
Duplicate of https://github.com/microsoft/CCF/pull/1614#discussion_r490070508.